barretenberg/cpp/src/barretenberg/ultra_honk/ultra_composer.hpp

```diff
@@ -83,33 +81,6 @@ template <UltraFlavor Flavor> class UltraComposer_ {
 
     UltraVerifier_<Flavor> create_ultra_with_keccak_verifier(CircuitBuilder& circuit);
 
-    /**
-     * @brief Create Prover for Goblin ECC op queue merge protocol
-     *
-     * @param op_queue
-     * @return MergeProver_<Flavor>
-     * TODO(https://github.com/AztecProtocol/barretenberg/issues/804): Goblin should be responsible for constructing
-     * merge prover/verifier.
-     */
-    MergeProver_<Flavor> create_merge_prover(
-        const std::shared_ptr<ECCOpQueue>& op_queue,
-        const std::shared_ptr<Transcript>& transcript = std::make_shared<Transcript>())
-    {
-        // Store the previous aggregate op queue size and update the current one
```

Comment:
I think the comments here are valuable and are not entirely reflected in the changes to MergeProver

Reply:
Can you be more specific? Looks like there are two comments here and I think they both made it into the new MergeProver constructor (albeit in a slightly modified form)

Reply:
yes, indeed, I rechecked, my bad